web_app/dist/title.css

```diff
@@ -0,0 +1,57 @@
+@import url('https://fonts.googleapis.com/css?family=Raleway');
+
+html {
+    
+}
+
+body {
+    background: #93ca68;
+    padding: 70px 0;
+    text-align: center;
+    color: white;
+    font-family: 'Raleway', sans-serif;
+    vertical-align: middle;
+}
+
+.title-page-font {
+    font-family: "Raleway", Helvetica, Arial, sans-serif;
+}
+
+.title-page-spacing {
+    padding: 0;
+    margin: 0;
+}
+
+#title{
+    font-size: 100px;
+}
+
+#tagline{
+    font-size: 30px;
+}
+
+.logoCenter {
+    position: absolute;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    opacity: 0.15;
+}
+
+.title-text-container {
+    position: absolute;
```

Comment:
Let's make an `absolute-center` class that contains the attributes shared between this and `.logo-center`

Reply:
done.